src/table/table-role/__integ__/grid-navigation.test.ts

```diff
@@ -29,8 +29,8 @@ const setupTest = (
 test(
   'cell action remains focused when row re-renders',
   setupTest({ actionsMode: 'inline' }, async page => {
-    await page.click('button[aria-label="Update item"]');
-    await expect(page.isFocused('button[aria-label="Update item"]')).resolves.toBe(true);
+    await page.click('button[aria-label="Update item"]:not([disabled])');
```

Comment:
In the newly generated data for table with grid navigation the first active "Update item" action moved to the 2nd row, so I updated the selector to ignore the prev row's disabled action.